<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Infrared Temperature Measurement</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QGroupBox" name="temperatureGroupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>Temperature</string>
          </property>
          <layout class="QGridLayout" name="gridLayout"/>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QGroupBox" name="groupBox_4">
            <property name="title">
             <string>EEPROM</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="0" column="0">
              <widget class="QTableWidget" name="EEPROMTableWidget">
               <attribute name="verticalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <row>
                <property name="text">
                 <string>1</string>
                </property>
               </row>
               <row>
                <property name="text">
                 <string>2</string>
                </property>
               </row>
               <row>
                <property name="text">
                 <string>3</string>
                </property>
               </row>
               <row>
                <property name="text">
                 <string>4</string>
                </property>
               </row>
               <row>
                <property name="text">
                 <string>6</string>
                </property>
               </row>
               <row>
                <property name="text">
                 <string>7</string>
                </property>
               </row>
               <row>
                <property name="text">
                 <string>8</string>
                </property>
               </row>
               <row>
                <property name="text">
                 <string>9</string>
                </property>
               </row>
               <row>
                <property name="text">
                 <string>10</string>
                </property>
               </row>
               <row>
                <property name="text">
                 <string>11</string>
                </property>
               </row>
               <row>
                <property name="text">
                 <string>13</string>
                </property>
               </row>
               <column>
                <property name="text">
                 <string>Address(Hex)</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Name</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Value(Hex)</string>
                </property>
               </column>
               <item row="0" column="0">
                <property name="text">
                 <string>000</string>
                </property>
               </item>
               <item row="0" column="1">
                <property name="text">
                 <string>TOMAX</string>
                </property>
               </item>
               <item row="0" column="2">
                <property name="text">
                 <string>0000</string>
                </property>
               </item>
               <item row="1" column="0">
                <property name="text">
                 <string>001</string>
                </property>
               </item>
               <item row="1" column="1">
                <property name="text">
                 <string>TOMIN</string>
                </property>
               </item>
               <item row="1" column="2">
                <property name="text">
                 <string>0000</string>
                </property>
               </item>
               <item row="2" column="0">
                <property name="text">
                 <string>002</string>
                </property>
               </item>
               <item row="2" column="1">
                <property name="text">
                 <string>PWMCTRL</string>
                </property>
               </item>
               <item row="2" column="2">
                <property name="text">
                 <string>0000</string>
                </property>
               </item>
               <item row="3" column="0">
                <property name="text">
                 <string>003</string>
                </property>
               </item>
               <item row="3" column="1">
                <property name="text">
                 <string>Ta range</string>
                </property>
               </item>
               <item row="3" column="2">
                <property name="text">
                 <string>0000</string>
                </property>
               </item>
               <item row="4" column="0">
                <property name="text">
                 <string>004</string>
                </property>
               </item>
               <item row="4" column="1">
                <property name="text">
                 <string>Emissivity</string>
                </property>
               </item>
               <item row="4" column="2">
                <property name="text">
                 <string>0000</string>
                </property>
               </item>
               <item row="5" column="0">
                <property name="text">
                 <string>005</string>
                </property>
               </item>
               <item row="5" column="1">
                <property name="text">
                 <string>Config Register1</string>
                </property>
               </item>
               <item row="5" column="2">
                <property name="text">
                 <string>0000</string>
                </property>
               </item>
               <item row="6" column="0">
                <property name="text">
                 <string>00E</string>
                </property>
               </item>
               <item row="6" column="1">
                <property name="text">
                 <string>SMBus address</string>
                </property>
               </item>
               <item row="6" column="2">
                <property name="text">
                 <string>0000</string>
                </property>
               </item>
               <item row="7" column="0">
                <property name="text">
                 <string>01C</string>
                </property>
               </item>
               <item row="7" column="1">
                <property name="text">
                 <string>ID number0</string>
                </property>
               </item>
               <item row="7" column="2">
                <property name="text">
                 <string>0000</string>
                </property>
               </item>
               <item row="8" column="0">
                <property name="text">
                 <string>01D</string>
                </property>
               </item>
               <item row="8" column="1">
                <property name="text">
                 <string>ID number1</string>
                </property>
               </item>
               <item row="8" column="2">
                <property name="text">
                 <string>0000</string>
                </property>
               </item>
               <item row="9" column="0">
                <property name="text">
                 <string>01E</string>
                </property>
               </item>
               <item row="9" column="1">
                <property name="text">
                 <string>ID number2</string>
                </property>
               </item>
               <item row="9" column="2">
                <property name="text">
                 <string>0000</string>
                </property>
               </item>
               <item row="10" column="0">
                <property name="text">
                 <string>01F</string>
                </property>
               </item>
               <item row="10" column="1">
                <property name="text">
                 <string>ID number3</string>
                </property>
               </item>
               <item row="10" column="2">
                <property name="text">
                 <string>0000</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_5">
            <property name="title">
             <string>RAM</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="0" column="0">
              <widget class="QTableWidget" name="RAMTableWidget">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>90</height>
                </size>
               </property>
               <attribute name="verticalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <row>
                <property name="text">
                 <string>1</string>
                </property>
               </row>
               <row>
                <property name="text">
                 <string>2</string>
                </property>
               </row>
               <row>
                <property name="text">
                 <string>3</string>
                </property>
               </row>
               <column>
                <property name="text">
                 <string>Address(Hex)</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Name</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Value(Hex)</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Temperature</string>
                </property>
               </column>
               <item row="0" column="0">
                <property name="text">
                 <string>006</string>
                </property>
               </item>
               <item row="0" column="1">
                <property name="text">
                 <string>TA</string>
                </property>
               </item>
               <item row="0" column="2">
                <property name="text">
                 <string>0000</string>
                </property>
               </item>
               <item row="0" column="3">
                <property name="text">
                 <string>0</string>
                </property>
               </item>
               <item row="1" column="0">
                <property name="text">
                 <string>007</string>
                </property>
               </item>
               <item row="1" column="1">
                <property name="text">
                 <string>TOBJ1</string>
                </property>
               </item>
               <item row="1" column="2">
                <property name="text">
                 <string>0000</string>
                </property>
               </item>
               <item row="1" column="3">
                <property name="text">
                 <string>0</string>
                </property>
               </item>
               <item row="2" column="0">
                <property name="text">
                 <string>008</string>
                </property>
               </item>
               <item row="2" column="1">
                <property name="text">
                 <string>TOBJ2</string>
                </property>
               </item>
               <item row="2" column="2">
                <property name="text">
                 <string>0000</string>
                </property>
               </item>
               <item row="2" column="3">
                <property name="text">
                 <string>0</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Device Index:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="deviceIndexComboBox">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>0</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>3</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>4</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>5</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>6</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>7</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>8</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>9</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Channel Index:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="channelIndexComboBox">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>0</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="startPushButton">
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>655</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
